docs/readme.md

```diff
@@ -0,0 +1 @@
+## Documentation can be found at https://onnxruntime.ai/docs
```

Comment:
my understanding is that onnxruntime.ai/docs has the user-facing documentation and this directory contains docs more relevant to ORT developers. if that's the case, could this wording be more specific?